.github/workflows/comment-artifacts.yml

```diff
@@ -0,0 +1,37 @@
+name: Leave comment with artifacts link 
+on:
+  workflow_run:
```

Comment:
Ohhhhh...

- https://github.community/t/workflow-run-not-working-as-expected/139342/2 / 
- https://github.com/github/docs/issues/799
- https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_run

> This event will only trigger a workflow run if the workflow file is on the default branch.

So we need to merge this to get it to run...

Reply:
So it may fail as our syntax for `runId` below is for this workflow, not the triggering workflow. But we can probably fix it in follow-up.